style.css

```diff
@@ -679,6 +679,140 @@ footer {
   display: none;
 }
 
+/* Popup desigining starts here !  */
+
+.popup_background_container {
+  background-color: rgb(0, 0, 0, 70%);
+  position: fixed;
+  top: 0;
+  width: 100vw;
+  height: 100vh;
+}
+
+.main_popup_container {
+  background-color: white;
+  position: fixed;
+  top: 0;
+  margin: 1em;
+  padding: 1em;
+  height: 100vh;
+  overflow: scroll;
+}
+
+.popup_heading_container {
+  display: flex;
+  justify-content: space-between;
+  font-size: 1.2rem;
+  font-weight: 700;
+  font-family: "Roboto", sans-serif;
+  margin-bottom: 1.5em;
+  align-items: center;
+}
+
+.popup_heading_container > span {
+  font-size: 2rem;
+  padding-right: 0.5em;
+}
+
+.popup_language_box {
+  list-style: none;
+  display: flex;
+  margin-bottom: 1.5em;
+}
+
+.language_box_item {
+  border: 0.5px solid #344563;
+  font-size: 0.8rem;
+  padding: 1em;
+  font-weight: 700;
+  font-family: "Roboto", sans-serif;
+  margin-right: 1rem;
+}
+
+.popup_img {
+  width: 100%;
+  margin-bottom: 1.5em;
+}
+
+.popup_para {
+  font-size: 1rem;
+  line-height: 1.8em;
+  font-family: "Inter", sans-serif;
+  font-weight: 400;
+  color: #344563;
+  margin-bottom: 1.5em;
+}
+
+.popup_button_main_container {
+  display: flex;
+  justify-content: space-between;
+}
+
+.popup_buttons_container {
+  background-color: #ff6b00;
+  display: flex;
+  align-items: center;
+  padding: 0.5em;
+}
+
+.popup_buttons {
+  font-size: 1.1rem;
+  font-weight: 700;
+  font-family: "Inter", sans-serif;
+  background: #ff6b00;
+  border: none;
+  color: white;
+  cursor: pointer;
+}
+
+.popup_icons {
+  margin-left: 1em;
+}
+
+/* responsive popup starts here */
+
+@media screen and (min-width: 789px) {
+  .popup_background_container {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+  }
+
+  .main_popup_container {
+    background-color: white;
+    width: 70%;
+    overflow: hidden;
+    top: 55px;
+  }
+
+  .popup_heading_container {
+    font-size: 2rem;
+    margin-bottom: 0.5em;
+  }
+
+  .language_box_item {
+    padding: 1em 2em;
+  }
+
+  .popup_grid_container {
+    display: grid;
+    grid-template-columns: 1.5fr 1fr;
+    grid-template-rows: 9fr 1fr;
+    column-gap: 25px;
+  }
+
+  .popup_img {
+    grid-row-start: 1;
+    grid-row-end: 3;
+    height: 100%;
+  }
+
+  .popup_button_main_container {
+    gap: 0.3em;
+    justify-content: flex-start;
+  }
+}
+
```

Comment:
- [ ] Kindly make sure your pop-up card styles match the provided Figma as much as possible and please make sure to adjust the background of the card to make sure it matches the provided design.
- [ ] Kindly make sure our pop-up mobile design matches the provided Figma as well.

   - ### YOUR DESIGN
   
    ![Screenshot from 2023-03-08 11-29-11](https://user-images.githubusercontent.com/87197412/223675951-2a6ab07a-547e-4dbd-9d81-5b39937a23df.png)

      

   - ### FIGMA DESIGN
   
  
    ![Screenshot from 2023-03-08 11-29-46](https://user-images.githubusercontent.com/87197412/223675764-143e33b6-773e-4652-9762-ed890aff99ea.png)

  ![image](https://user-images.githubusercontent.com/87197412/223676300-470b5787-1c4a-4047-91dc-913bffa33258.png)